olioapps/aframe-typescript-toolkit
# Purge dependencies

### WHAT 
- update toolkit to v. 0.0.17
- remove unused dependencies from toolkit, examples, and templates 
- updated examples READMEs, scripts, webpack 
- refactored examples to fit updated environment used in templates 
- rebuilt examples and toolkit

### TEST 
1. test examples
- pull branch `git pull && git checkout purge-dependencies`
-  run position logger example `cd examples/position_logger_component/  && yarn server`
- open http://localhost:3000/
- you should see this: 
![sphere](https://user-images.githubusercontent.com/15587168/48288878-75bad680-e422-11e8-999c-b3a192f0710e.gif)

- cancel server, run sphere system `cd ../sphere_registry_system/ && yarn server`
- open http://localhost:3000/
- you should see this 
![system](https://user-images.githubusercontent.com/15587168/48288884-7b182100-e422-11e8-88e5-45132b64ce07.gif)

2. test the whole toolkit 
- after reviewing merge v. 0.0.17 to master, push to npm and make sure it still works!

### todo list: 
https://trello.com/c/FRJSQ7HS/23-toolkit-release-nov-2018
